README.adoc

```diff
@@ -223,6 +223,34 @@ refute(object.empty?)
 refute_empty(object)
 ----
 
+=== Assert Empty Literal [[assert-empty-literal]]
+
+Prefer using `assert_empty(object)` over `assert([], object)` or `assert({}, object)`.
+
+[source,ruby]
+----
+# bad
+assert([], object)
```

Comment:
@tejasbubane `assert` only accepts the first argument as an expectation. The second argument is considered as a custom message. Ref: https://github.com/seattlerb/minitest/blob/master/lib/minitest/assertions.rb#L178

So, I tried it on my local. 
```
assert([], object) => Passes
assert([], [2]) => Passes
assert([], User.first) => Passes
```

Just for my information, can you share an example where Empty literal are tested? I normally have used `assert_empty` or `assert(object.empty?)` but never wrote a spec for `assert([])` empty literal.

Reply:
That's exactly the issue here: https://github.com/rubocop-hq/rubocop-minitest/issues/32

And now that I think of it, this does not fit well in the style guide because it is very less likely to be used like this. 

Should we change it to `assert_equal` instead of `assert`. And maybe the cop can handle both `assert` (as per bug) and `assert_equal` (most likely to be used.

Reply:
I am not sure how the node matcher would be written when `assert` and `assert_equal` have different arguments expectations and `assert` expecting the second argument as Array or Hash seems wrong to me. WDYT? 

I would suggest lets only support `assert_equal` and not `assert`.